### ISSUE TITLE
PA REVIEW: Lists

### ISSUE THREAD

**Issue by walterhwan** [walterhwan/remember_the_bacon#6]
- [x] User can create/edit/delete list
- [x] Adequate styling
- [x] Smooth, bug-free navigation
- [x] Adequate and appropriate seeds to demonstrate the feature

**Comment by AAlfarho** [comment]
Good handling list removal, specifically when the tasks of such list are being displayed, kudos!
There are however a couple of issues we need to address before we can consider this feature to be completed as MVP.

- [x] After creating a new task and hovering over it, we don't have the same style the rest of the lists.
   + Existing list
          ![other](https://user-images.githubusercontent.com/1103663/38632562-f7b68d4e-3d71-11e8-9748-4cff0e6ce7cf.png)
   + New list
          ![custom](https://user-images.githubusercontent.com/1103663/38632582-05a46afc-3d72-11e8-94d7-fc5d22ee1de3.png)
- [x] While hovering over a list, the `configuration cog` is not centered with the current list, and is not aligned with the `+`.
     + ![list_cog](https://user-images.githubusercontent.com/1103663/38632756-81258602-3d72-11e8-93f5-ba9c21563c80.png)
- [x] When removing a list, a modal should pop-up and warn us if we are sure we want to remove the list.
- [x] When renaming a list, a model shows up, modals by default have a specific behavior, if you hit `esc` they should close themselves, by hitting `enter` they should submit their action, let's implement this. This also applies to the `Warning modal` for delete.

Feel free to mark the checkboxes yourself after addressing them. Assign this issue back to me once the updates are live in Heroku.

**Comment by walterhwan** [comment]
About this issue

> While hovering over a list, the configuration cog is not centered with the current list, and is not aligned with the +.

I centered with the current list. But the none-aligenment with the `+` icon is following the original design of the website

**Comment by walterhwan** [comment]
About 

> When renaming a list, a model shows up, modals by default have a specific behavior, if you hit esc they should close themselves, by hitting enter they should submit their action, let's implement this. This also applies to the Warning modal for delete.

The original website don't have esc close modal functionality. But I added it anyway.(but you have to hit it twice, once is escaping input field once is closing the modal)  Also, I have the submit form when enter already. Maybe at the time you saw it I haven't upload it to heroku.

**Comment by AAlfarho** [comment]
> The original website don't have esc close modal functionality.
They do have `esc` functionality for the update and remove list  _modals_.

> But the none-aligenment with the + icon is following the original design of the website
Fair point.

This concludes the feedback for the lists MVP, kudos!
